Karma/Areas/Identity/Pages/Account/Manage/History.cshtml

```diff
@@ -27,20 +27,24 @@
 
             <div class="tab-content" id="pills-tabContent">
                 <div class="tab-pane fade show active" id="pills-a" role="tabpanel" aria-labelledby="pills-a-tab">
-                    @{await (Html.RenderPartialAsync("~/Areas/Identity/Pages/Account/Manage/HistoryTable.cshtml", await Model.UserPosts.Value));}
+                    @{await (Html.RenderPartialAsync("~/Areas/Identity/Pages/Account/Manage/HistoryTable.cshtml", 
+                          (await Model.UserPosts.Value).Where(p => p.UserId == User.Identity?.Name)));}
```

Comment:
If the user has received an item during a trade (he wasn't the one that posted the item), the trade isn't shown in `All listings`

The received item:
![Screenshot 2021-11-14 at 13 32 13](https://user-images.githubusercontent.com/73688133/141679290-c36f461a-6eee-4abd-9a06-4e6324fa5f08.png)

The post does not appear between `All listings`:
![Screenshot 2021-11-14 at 13 32 03](https://user-images.githubusercontent.com/73688133/141679303-9cf70792-6787-4c40-993a-c64092f89e52.png)

Reply:
Is it really logical to put someone's else listing in your listings list?

Reply:
Maybe it should be called 'Uploaded posts' or something like that?
I thought it's 'All listings' because it's supposed to be showing the whole trading and posting history of that person.

Reply:
Good point, I'll rename it.